Initial implementation of Reference Material page

jreyno77 opened the pull request.

JPercival commented:
Looks good. Can we removed the CQL menu item so that's it's not duplicated across reference material in the menu? And once we've done that can we update the Getting Started page to have a link to the reference material?

rob-reynolds commented:
Oh, that "CQL specification" menu item is, itself, a Reference Material (and is included in the RM page).

I completely missed that in the requirements description.

jreyno77 commented:
I removed the CQL menu item and deleted the link for it in the Getting Started page.  I will work on adding the link to reference material later today
